learning-objectives/data.yml

```
@@ -58,6 +58,73 @@ js:
     # - async/await
     # - streams
 
+java:
+  - data-types:
+      - primitive
+      - primitive-vs-non-primitive
+      - strings
+      - arrays
+      - void
+      - enums
+      - date
+  - access-modifiers:
+      - private
+      - protected
+      - public 
+  - object-oriented:
+      - classes-and-objects:
+        - classes
+        - objects
+        - methods
+        - attributes
+        - constructors
+        - encapsulation
+      - abstraction
+      - interface
+      - polimorphism
+      - inherit:
+        - superclass
+        - extends
+        - override
+  - variables
+  - conditionals
+  - loops
+  - lambda
+  - optional
+  - stream
+  - collections:
+    - list:
+      - arraylist
+    - set:
+      - hashset
+    - map:
+      - hashmap
+  - exceptions
+
+hibernate: 
+  - setup
+  - orm:
+    - entity
+    - table
+    - columns
+    - relationship:
+      - manytoone
+      - onetomany
+      - manytomany
+  - create
+  - read
+  - update
+  - delete
+
+spring: 
+  - setup:
+    - initializr
+    - gradle
+  - spring-jpa
+  - spring-web
+  - spring-security
```

spring
```suggestion
spring:
  - core-concepts:
    - beans
    - inversion-of-control
    - annotations
  - spring-boot:
    - servlets
    - controllers
    - services
  - spring-data-jpa:
    - entities
    - repository
  - setup:
    - initializr
    - gradle
```
@NaraSakamoto que te parece esto?
spring-security no lo estamos contemplando

@NaraSakamoto O que você acha disso?
Não estamos contemplando spring-security

---

@merunga Gostei das sugestões! Faz muito sentido o core-concepts. Tenho dúvidas somente sobre servlets. Acho que nunca cheguei a usar com o spring-boot! Será que é algo mais recente?  Acho que valeria adicionar componets, também! Junto de services e controllers são annotations muito utilizadas.